wpiutil/src/main/native/cpp/llvm/ErrorHandling.cpp

```diff
@@ -15,7 +15,6 @@
 #include "wpi/ErrorHandling.h"
 #include "wpi/SmallVector.h"
 #include "wpi/Twine.h"
-#include "wpi/Error.h"
```

Comment:
The llvm files shouldn't be touched, we occasionally sync with upstream and changing these makes merging harder.